environment.yml

```diff
@@ -18,7 +18,6 @@ dependencies:
  - pytest-runner
  - spyder
  - pyzmq
- - ruamel_yaml
```

Comment:
we don't need to include it in the "pip" section becuase it's a requirement in qcodes's setup file, right?

Reply:
No but that is true for a lot of package in both the env file and the requirements